content/docs/api-reference/get_params.md

```diff
@@ -0,0 +1,90 @@
+# dvc.api.get_params()
+
+Returns all the <abbr>params</abbr> associated with the given
```

Comment:
does it make sense to return all params by default and make stage name an optional target?

Reply:
That kind of deviates from the use case I had in mind for this API, which is retrieving only the parameters relevant to the current stage being run.

Retrieving all DVC params is, IMO, a different use case and can be done with `dvc.api.Repo().params.show()`. Would a note with this snippet be enough?

Reply:
I see what you mean regarding the use case, but since we are adding this as a public interface we should be also thinking about some general scenarios. In this case it feels natural to expect that target is optional (DVC CLI does the same in a lot of cases), but since it can be changed later (almost) w/o breaking compatibility we can go like this for now. Your call. (I would not include a snippet though with Repo API - it's not public yet?)